ERROR - 2024-01-26 01:08:02 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\POS_CODEIGNITER\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\POS_CODEIGNITER\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'db_pos', 3306, '', 0)
#1 C:\xampp\htdocs\POS_CODEIGNITER\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\POS_CODEIGNITER\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\POS_CODEIGNITER\system\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\POS_CODEIGNITER\system\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\POS_CODEIGNITER\system\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('dedi', 'username', Array, NULL, 'username')
#6 C:\xampp\htdocs\POS_CODEIGNITER\system\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('username', 'username', 'dedi', Array, Array)
#7 C:\xampp\htdocs\POS_CODEIGNITER\system\Controller.php(152): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\POS_CODEIGNITER\app\Controllers\Auth.php(26): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\POS_CODEIGNITER\system\CodeIgniter.php(941): App\Controllers\Auth->login()
#10 C:\xampp\htdocs\POS_CODEIGNITER\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#11 C:\xampp\htdocs\POS_CODEIGNITER\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\POS_CODEIGNITER\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\POS_CODEIGNITER\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\POS_CODEIGNITER\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\POS_CODEIGNITER\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\POS_CODEIGNITER\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\POS_CODEIGNITER\system\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\POS_CODEIGNITER\system\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\POS_CODEIGNITER\system\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('dedi', 'username', Array, NULL, 'username')
#5 C:\xampp\htdocs\POS_CODEIGNITER\system\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('username', 'username', 'dedi', Array, Array)
#6 C:\xampp\htdocs\POS_CODEIGNITER\system\Controller.php(152): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\POS_CODEIGNITER\app\Controllers\Auth.php(26): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\POS_CODEIGNITER\system\CodeIgniter.php(941): App\Controllers\Auth->login()
#9 C:\xampp\htdocs\POS_CODEIGNITER\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 C:\xampp\htdocs\POS_CODEIGNITER\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\POS_CODEIGNITER\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\POS_CODEIGNITER\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-01-26 01:08:02 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `tb_users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('dedi', 'username', [...], null, 'username')
 5 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('username', 'username', 'dedi', [...], [...])
 6 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\Auth.php(26): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
ERROR - 2024-01-26 01:18:21 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\POS_CODEIGNITER\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\POS_CODEIGNITER\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'db_pos', 3306, '', 0)
#1 C:\xampp\htdocs\POS_CODEIGNITER\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\POS_CODEIGNITER\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\POS_CODEIGNITER\system\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\POS_CODEIGNITER\system\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\POS_CODEIGNITER\system\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin01', 'username', Array, NULL, 'username')
#6 C:\xampp\htdocs\POS_CODEIGNITER\system\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('username', 'username', 'admin01', Array, Array)
#7 C:\xampp\htdocs\POS_CODEIGNITER\system\Controller.php(152): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\POS_CODEIGNITER\app\Controllers\Auth.php(26): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\POS_CODEIGNITER\system\CodeIgniter.php(941): App\Controllers\Auth->login()
#10 C:\xampp\htdocs\POS_CODEIGNITER\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#11 C:\xampp\htdocs\POS_CODEIGNITER\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\POS_CODEIGNITER\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\POS_CODEIGNITER\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\POS_CODEIGNITER\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\POS_CODEIGNITER\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\POS_CODEIGNITER\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\POS_CODEIGNITER\system\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\POS_CODEIGNITER\system\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\POS_CODEIGNITER\system\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin01', 'username', Array, NULL, 'username')
#5 C:\xampp\htdocs\POS_CODEIGNITER\system\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('username', 'username', 'admin01', Array, Array)
#6 C:\xampp\htdocs\POS_CODEIGNITER\system\Controller.php(152): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\POS_CODEIGNITER\app\Controllers\Auth.php(26): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\POS_CODEIGNITER\system\CodeIgniter.php(941): App\Controllers\Auth->login()
#9 C:\xampp\htdocs\POS_CODEIGNITER\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 C:\xampp\htdocs\POS_CODEIGNITER\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\POS_CODEIGNITER\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\POS_CODEIGNITER\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-01-26 01:18:21 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `tb_users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(227): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(328): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin01', 'username', [...], null, 'username')
 5 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('username', 'username', 'admin01', [...], [...])
 6 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\Auth.php(26): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
CRITICAL - 2024-01-26 01:46:18 --> ErrorException: Undefined variable: message
in APPPATH\Controllers\Auth.php on line 52.
 1 APPPATH\Controllers\Auth.php(52): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: message', 'C:\\xampp\\htdocs\\POS_CODEIGNITER\\app\\Controllers\\Auth.php', 52, [...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
CRITICAL - 2024-01-26 01:48:04 --> ErrorException: Undefined variable: message
in APPPATH\Controllers\Auth.php on line 52.
 1 APPPATH\Controllers\Auth.php(52): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: message', 'C:\\xampp\\htdocs\\POS_CODEIGNITER\\app\\Controllers\\Auth.php', 52, [...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
CRITICAL - 2024-01-26 01:49:51 --> ErrorException: Undefined variable: message
in APPPATH\Controllers\Auth.php on line 52.
 1 APPPATH\Controllers\Auth.php(52): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: message', 'C:\\xampp\\htdocs\\POS_CODEIGNITER\\app\\Controllers\\Auth.php', 52, [...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
CRITICAL - 2024-01-26 01:51:38 --> ErrorException: Undefined variable: message
in APPPATH\Controllers\Auth.php on line 52.
 1 APPPATH\Controllers\Auth.php(52): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: message', 'C:\\xampp\\htdocs\\POS_CODEIGNITER\\app\\Controllers\\Auth.php', 52, [...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
CRITICAL - 2024-01-26 01:54:59 --> ErrorException: Undefined variable: message
in APPPATH\Controllers\Auth.php on line 52.
 1 APPPATH\Controllers\Auth.php(52): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: message', 'C:\\xampp\\htdocs\\POS_CODEIGNITER\\app\\Controllers\\Auth.php', 52, [...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
CRITICAL - 2024-01-26 02:00:18 --> ErrorException: Undefined variable: message
in APPPATH\Controllers\Auth.php on line 52.
 1 APPPATH\Controllers\Auth.php(52): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: message', 'C:\\xampp\\htdocs\\POS_CODEIGNITER\\app\\Controllers\\Auth.php', 52, [...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
CRITICAL - 2024-01-26 02:03:17 --> ErrorException: Undefined variable: message
in APPPATH\Controllers\Auth.php on line 52.
 1 APPPATH\Controllers\Auth.php(52): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: message', 'C:\\xampp\\htdocs\\POS_CODEIGNITER\\app\\Controllers\\Auth.php', 52, [...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
CRITICAL - 2024-01-26 02:03:47 --> ErrorException: Undefined variable: message
in APPPATH\Controllers\Auth.php on line 52.
 1 APPPATH\Controllers\Auth.php(52): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: message', 'C:\\xampp\\htdocs\\POS_CODEIGNITER\\app\\Controllers\\Auth.php', 52, [...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
CRITICAL - 2024-01-26 02:10:45 --> Error: Call to undefined function form_open()
in APPPATH\Views\auth\login.php on line 37.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('auth/login', [], true)
 4 APPPATH\Controllers\Auth.php(17): view('auth/login', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
CRITICAL - 2024-01-26 02:11:17 --> Error: Call to undefined function form_open()
in APPPATH\Views\auth\login.php on line 37.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('auth/login', [], true)
 4 APPPATH\Controllers\Auth.php(17): view('auth/login', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
CRITICAL - 2024-01-26 02:11:29 --> Error: Call to undefined function form_open()
in APPPATH\Views\auth\login.php on line 37.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('auth/login', [], true)
 4 APPPATH\Controllers\Auth.php(17): view('auth/login', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
CRITICAL - 2024-01-26 02:18:05 --> ErrorException: Undefined variable: message
in APPPATH\Controllers\Auth.php on line 52.
 1 APPPATH\Controllers\Auth.php(52): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: message', 'C:\\xampp\\htdocs\\POS_CODEIGNITER\\app\\Controllers\\Auth.php', 52, [...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
CRITICAL - 2024-01-26 02:18:55 --> ErrorException: Undefined variable: message
in APPPATH\Controllers\Auth.php on line 52.
 1 APPPATH\Controllers\Auth.php(52): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: message', 'C:\\xampp\\htdocs\\POS_CODEIGNITER\\app\\Controllers\\Auth.php', 52, [...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
CRITICAL - 2024-01-26 02:18:57 --> ErrorException: Undefined variable: message
in APPPATH\Controllers\Auth.php on line 52.
 1 APPPATH\Controllers\Auth.php(52): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: message', 'C:\\xampp\\htdocs\\POS_CODEIGNITER\\app\\Controllers\\Auth.php', 52, [...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
CRITICAL - 2024-01-26 02:19:08 --> ErrorException: Undefined variable: message
in APPPATH\Controllers\Auth.php on line 52.
 1 APPPATH\Controllers\Auth.php(52): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: message', 'C:\\xampp\\htdocs\\POS_CODEIGNITER\\app\\Controllers\\Auth.php', 52, [...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
CRITICAL - 2024-01-26 02:20:47 --> ErrorException: Undefined variable: message
in APPPATH\Controllers\Auth.php on line 52.
 1 APPPATH\Controllers\Auth.php(52): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: message', 'C:\\xampp\\htdocs\\POS_CODEIGNITER\\app\\Controllers\\Auth.php', 52, [...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
CRITICAL - 2024-01-26 03:07:25 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "layouts/sidebar.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(216): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('layouts/sidebar.php')
 2 SYSTEMPATH\View\View.php(469): CodeIgniter\View\View->render('layouts/sidebar', null, true)
 3 APPPATH\Views\layouts\maintemplate.php(35): CodeIgniter\View\View->include('layouts/sidebar')
 4 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\POS_CODEIGNITER\\app\\Views\\layouts\\maintemplate.php')
 5 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\View\View.php(244): CodeIgniter\View\View->render('layouts/maintemplate', [], true)
 7 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('welcome_message', [], true)
 8 APPPATH\Controllers\Home.php(9): view('welcome_message')
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->index()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
CRITICAL - 2024-01-26 07:46:53 --> CodeIgniter\HTTP\Exceptions\HTTPException: Failed to parse JSON string. Error: Syntax error
in SYSTEMPATH\HTTP\IncomingRequest.php on line 588.
 1 SYSTEMPATH\HTTP\IncomingRequest.php(588): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidJSON('Syntax error')
 2 SYSTEMPATH\HTTP\IncomingRequest.php(608): CodeIgniter\HTTP\IncomingRequest->getJSON(true)
 3 SYSTEMPATH\HTTP\IncomingRequest.php(556): CodeIgniter\HTTP\IncomingRequest->getJsonVar('username', false, null, null)
 4 APPPATH\Controllers\Auth.php(22): CodeIgniter\HTTP\IncomingRequest->getVar('username')
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
CRITICAL - 2024-01-26 07:50:07 --> CodeIgniter\HTTP\Exceptions\HTTPException: Failed to parse JSON string. Error: Syntax error
in SYSTEMPATH\HTTP\IncomingRequest.php on line 588.
 1 SYSTEMPATH\HTTP\IncomingRequest.php(588): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidJSON('Syntax error')
 2 SYSTEMPATH\HTTP\IncomingRequest.php(608): CodeIgniter\HTTP\IncomingRequest->getJSON(true)
 3 SYSTEMPATH\HTTP\IncomingRequest.php(556): CodeIgniter\HTTP\IncomingRequest->getJsonVar('username', false, null, null)
 4 APPPATH\Controllers\Auth.php(22): CodeIgniter\HTTP\IncomingRequest->getVar('username')
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
CRITICAL - 2024-01-26 07:50:27 --> CodeIgniter\HTTP\Exceptions\HTTPException: Failed to parse JSON string. Error: Syntax error
in SYSTEMPATH\HTTP\IncomingRequest.php on line 588.
 1 SYSTEMPATH\HTTP\IncomingRequest.php(588): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidJSON('Syntax error')
 2 SYSTEMPATH\HTTP\IncomingRequest.php(608): CodeIgniter\HTTP\IncomingRequest->getJSON(true)
 3 SYSTEMPATH\HTTP\IncomingRequest.php(556): CodeIgniter\HTTP\IncomingRequest->getJsonVar('username', false, null, null)
 4 APPPATH\Controllers\Auth.php(22): CodeIgniter\HTTP\IncomingRequest->getVar('username')
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
CRITICAL - 2024-01-26 07:51:50 --> CodeIgniter\HTTP\Exceptions\HTTPException: Failed to parse JSON string. Error: Syntax error
in SYSTEMPATH\HTTP\IncomingRequest.php on line 588.
 1 SYSTEMPATH\HTTP\IncomingRequest.php(588): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidJSON('Syntax error')
 2 SYSTEMPATH\HTTP\IncomingRequest.php(608): CodeIgniter\HTTP\IncomingRequest->getJSON(true)
 3 SYSTEMPATH\HTTP\IncomingRequest.php(556): CodeIgniter\HTTP\IncomingRequest->getJsonVar('username', false, null, null)
 4 APPPATH\Controllers\Auth.php(22): CodeIgniter\HTTP\IncomingRequest->getVar('username')
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Auth->login()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\POS_CODEIGNITER\\public\\index.php')
